### PR TITLE
Rant on Contractors

### PR DESCRIPTION
**Pull request by Hamo69**
Draft 1 of my rant on consultants v contractors.

### CODE REVIEW

**Comment by Hamo69** [comment]
@temyers shall I kill this off or merge?

**Comment by temyers** [comment]
Haha, I'll let you decide.

**Review by temyers — DISMISSED** [review]

File: _posts/2020-09-23-these-are-not-the-consultants-you-are-looking-for.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+tags: consultants contractors consulting contracting rant
+author: Hamish Tedeschi
+image: img/consultants.jpeg
+---
+
+![Consultants](/img/consultants.jpeg)
+
+<br/>
+
+# Warning: this post contains a number of opinionated hot takes formed over the last 20 years of running companies in various countries and contracting or consulting in these countries. If this is not the sort of post you are looking for, then look away now..
```
Comment (temyers) [inline _posts/2020-09-23-these-are-not-the-consultants-you-are-looking-for.md:14]: This is H1 - suggest it's just BOLD text

**Review by temyers — APPROVED** [review]
I will no doubt regret this decision :facepalm:

**Pull request merged**